pandas/tests/io/formats/test_format.py

```diff
@@ -1451,6 +1451,11 @@ def test_to_string_format_na(self):
                     '4  4.0     bar')
         assert result == expected
 
+    def test_to_string_decimal(self):
+        df = DataFrame({'A': [6.0, 3.1, 2.2]})
```

Comment:
can you add the issue number

Reply:
Small thing, but I think we consistently use the format `GH 23614`. 

@gfyoung I think you changed some in a recent PR, can you confirm?